dbt_subprojects/dex/models/automated_trades/dex_automated_trades_all.sql

```diff
@@ -1,11 +1,11 @@
 {{ config(
     schema = 'dex'
-    , alias = 'automated_trades'
-    , partition_by = ['block_month', 'blockchain', 'project']
+    , alias = 'automated_trades_all'
+    , partition_by = ['blockchain', 'block_date']
     , materialized = 'incremental'
     , file_format = 'delta'
     , incremental_strategy = 'merge'
-    , unique_key = ['blockchain', 'project', 'version', 'tx_hash', 'evt_index']
+    , unique_key = ['blockchain', 'block_number', 'tx_index', 'evt_index']
```

Comment:
```suggestion
    , unique_key = ['blockchain', 'block_date', 'block_number', 'tx_index', 'evt_index']
```

Reply:
might be worth having `block_date` in here since it's part of the partition, should help join from source to target in merge statement